scripts/qa_crowbarsetup.sh

```diff
@@ -2963,6 +2963,9 @@ function deploy_single_proposal()
                 fi
             fi
             if iscloudver 6M9plus ; then
+                # we skip all the rest of the manila setup in case of an upgrade
+                # as the images and sec-groups are already in place
+                [[ -n $upgrade_cloudsource ]] && continue
```

Comment:
I agree that we would actually need the other checks. The thing is, you're assuming that what is done here will be the exact same code path for all cloud versions.

But: this is already not true for a 5->6 upgrade (since we only run this code for Cloud 6), and in Cloud 7, there might be a different setup to do for manila.

So I'm not really sure it's fully correct here. What's the intent behind skipping things in case of upgrade?